packages/synapse-react-client/src/lib/containers/AccessRequirementListV2/RequirementItem/SelfSignAccessRequirementItem.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react'
+import MarkdownSynapse from '../../markdown/MarkdownSynapse'
+import {
+  AccessApproval,
+  ApprovalState,
+  isTermsOfUseAccessRequirement,
+  SelfSignAccessRequirement,
+  TermsOfUseAccessRequirement,
+} from '../../../utils/synapseTypes'
+import { useSynapseContext } from '../../../utils/SynapseContext'
+import { Box, ButtonProps, Link, Typography } from '@mui/material'
+import {
+  useCreateAccessApproval,
+  useGetAccessRequirementStatus,
+  useGetAccessRequirementWikiPageKey,
+  useGetCurrentUserBundle,
+  useGetCurrentUserProfile,
+} from '../../../utils/hooks/SynapseAPI'
+import RequirementItem from './RequirementItem'
+import { RequirementItemState } from '../AccessApprovalCheckMarkV2'
+
+export type SelfSignAccessRequirementItemProps = {
+  accessRequirement: SelfSignAccessRequirement | TermsOfUseAccessRequirement
+  onHide: () => void
+}
+
+/**
+ * Renders a {@link RequirementItem} for an self-sign Access Requirement, including {@link SelfSignAccessRequirement} or
+ * {@link TermsOfUseAccessRequirement}.
+ *
+ * For {@link ManagedACTAccessRequirement}s, see {@link RequestDataAccess}
+ */
+export default function SelfSignAccessRequirementItem(
```

Comment:
Previous analog also handled (Unmanaged) `ACTAccessRequirement`, but I split that out into a different component in my implementation to reduce complexity.